image/templates/helm/stackrox-secured-cluster/internal/defaults/70-scanner-v4.yaml.htpl

```diff
@@ -1,6 +1,6 @@
 [<- if and .FeatureFlags.ROX_SCANNER_V4_SUPPORT (not .KubectlOutput) >]
 scannerV4:
-  disable: true [</* On secured-clusters local scanners (irregardless of the version) require explicit activation. */>]
```

Comment:
Removed to be able to notice when this field has not been explicitly set by the user.

Reply:
please resolve when noticed.